app/containers/paperShow/index.tsx

```diff
@@ -332,11 +371,40 @@ class PaperShow extends React.PureComponent<PaperShowProps, PaperShowStates> {
         <div className={styles.footerWrapper}>
           <Footer />
         </div>
-        <NextPaperTab paperId={paper.id} />
+        <NextPaperTab paperList={relatedPaperList} />
       </>
     );
   }
 
+  private setIsDownloadedPDF = (isDownloaded: boolean) => {
+    this.setState(prevState => ({ ...prevState, isDownloadedPDF: isDownloaded }));
+  };
+
+  private fetchRelatedPaperData = (paperId: number) => {
+    PaperAPI.getRelatedPapers({ paperId, cancelToken: this.cancelToken.token })
```

Comment:
I think this logic should be another Redux structure because it affects multiple components.
check this [post](https://github.com/pluto-net/scinapse-web-client/wiki/Redux-Structure-vs-Component-States-in-Pluto-prject) and let's make it together.
(or you can make it alone! just make another reducer isn't below PaperShow reducer.